content/radars/2021-09-devsecops.yml

```
@@ -0,0 +1,203 @@
+
+      Although the security portion of DevSecOps is pervasive, with many tools targeting security, the Radar team felt that many of these have to be stitched together within end user’s tech stacks. There is no tool, yet, that creates a smooth developer experience.
+
+      However, the growing noise in this space could prove beneficial for developers, with newcomer tools showing promising features.
+
+  - headline: The pace of change in the security space is rapid
+    content: |
+      More and more new tools are coming to the market to support DevSecOps, but practitioners are struggling to keep up with the ever-changing landscape and evaluate what tools are best for their needs.
+
+      Despite many years’ experience, the Radar team hadn’t heard of all of the listed tools, underscoring the rapid speed of change in this space.
+
+
+  - headline: Microsegmentation capability is very important, but presents a significant challenge
+    content: |
+      Many organizations are unable to successfully operationalize segmentation within cloud native environments. While many organizations have introduced robust DevOps processes, traditional network security practices have not kept pace with the shift to cloud native. 
+
+      The Radar team felt that tools like Calico and Cilium, which offer micro-segmentation capabilities at Layer 3-4, are becoming increasingly important in the DevSecOps space, alongside Layer 7 segmentation, with mesh technologies like Istio and Linkerd.
+
+      With the core focus on security, tools like Artifactory, SonarCube, Xray, and even GitHub, bring a mutual Transport Security Layer to users’ tech stacks. The Rader team also felt that DevSecOps practitioners are looking at a hardening of assets, rather than introducing more new tools, to meet security requirements.
+
+video: https://www.youtube.com/embed/dQw4w9WgXcQ
+team:
+  - name: Katie Gamanji
+    photo: https://www.cncf.io/wp-content/uploads/2020/11/ELxkBsqUUAEmp0h.jpeg
+    bio: |
+      Katie's mission is to make end users successful and productive with cloud native technologies such as Kubernetes and Prometheus.
+    twitter: k_gamanji
+    linkedin: katie-gamanji
+    title: Ecosystem Advocate at CNCF
+  - name: Sergiu Petean
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Sergio-Patean.jpeg
+    twitter: PeteanuSergiu
+    linkedin: peteanusergiu
+    title: Head of DevOps at Allianz Direct
+  - name: Keith Nielsen
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Keith-Nielsen.jpeg
+    title: Director Cloud Architecture at Discover Financial Services
+points:
+  - name: Artifactory 
+    homepage: https://jfrog.com/artifactory/
+    level: adopt
+    votes: 16
+      adopt: 14
+      trial: 1
+      hold: 1
```

We do not show the number of `hold` votes. 
```suggestion
      asses: ??
```

Tho looking at other radars, we seem to have the `hold` votes for some tools. Maybe we could add them for each tool in GitHub, as they will not be showcased on the radar page.

---

resolved on slack